### PR TITLE
Adding documentation for nuget commands in C#DK

### PR DESCRIPTION
**Pull request by albarry4**
Changing the package management file to add information about the new NuGet experience in C# Dev Kit

### CODE REVIEW

File: docs/csharp/package-management.md [hunk 1]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
 
 # NuGet in Visual Studio Code
 
-NuGet is the package manager for .NET. It defines how packages for .NET are created, hosted, and consumed, and provides the tools for each of those roles. Importantly, NuGet also manages the dependency tree on behalf of a project, so you only need to focus on the packages that you're directly using in a project. For more information, go to the [NuGet documentation](https://learn.microsoft.com/nuget/what-is-nuget).
+NuGet is the package manager for .NET. It defines how packages for .NET are created, hosted, and consumed, and provides the tools for each of those roles. You can manage your NuGet packages directly from the C# Dev Kit Solution explorer and commands in the command palette. NuGet also manages the dependency tree on behalf of a project, so you only need to focus on the packages that you're directly using in a project. For more information, go to the [NuGet documentation](https://learn.microsoft.com/nuget/what-is-nuget).
```
Comment (ntrogh) [inline docs/csharp/package-management.md:13]:
```suggestion
NuGet is the package manager for .NET. It defines how packages for .NET are created, hosted, and consumed, and provides the tools for each of those roles. You can manage your NuGet packages directly from the C# Dev Kit Solution explorer and commands in the Command Palette. NuGet also manages the dependency tree on behalf of a project, so you only need to focus on the packages that you're directly using in a project. For more information, go to the [NuGet documentation](https://learn.microsoft.com/nuget/what-is-nuget).
```

File: docs/csharp/package-management.md [hunk 2]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+
+* [NuGet Commands in C# Dev Kit](#nuget-commands-in-c-dev-kit)
+* [Dependency Management](#dependency-management)
+
+## NuGet Commands in C# Dev Kit
+
+In C# Dev Kit, you can perform NuGet package operations in either of two ways: commands in the command palette or through right-click actions in the C# Dev Kit solution explorer. The below documentation details how to use each NuGet command through both of the two methods.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:20]:
```suggestion
In C# Dev Kit, you can perform NuGet package operations in either of two ways: commands in the Command Palette or through right-click actions in the C# Dev Kit Solution explorer. The below documentation details how to use each NuGet command through both of the two methods.
```

File: docs/csharp/package-management.md [hunk 3]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+
+### Add a package
+
+To add a NuGet package to your project, use the command “NuGet: Add NuGet Package”.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:24]:
```suggestion
To add a NuGet package to your project, use the command **NuGet: Add NuGet Package**.
```

File: docs/csharp/package-management.md [hunk 4]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+
+![alt text](images/package-management/nuget-command-addpackage.png)
```
Comment (ntrogh) [inline docs/csharp/package-management.md:26]: Can you add a descriptive alt text for users that rely on it?

File: docs/csharp/package-management.md [hunk 5]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+
+Next, select the version you want to apply.
+
+![alt text](images/package-management/nuget-command-addpackage-versionselection.png)
+
+Once you select a package and version number, C# Dev Kit will add it to your project and update your project file and references.
+
+### Update a package
+
+To update a NuGet package in your project, use the command “NuGet: Update NuGet Package”. If you have more than one project in your solution, you will be asked to select which project contains the package that you would like to update. Next, you will see a list of the packages that are currently installed in your project which have an update available. Choose which one you want to update. If no packages in your solution have an update available, you will see a message notifying you of this.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:46]:
```suggestion
To update a NuGet package in your project, use the command **NuGet: Update NuGet Package**. If you have more than one project in your solution, you will be asked to select which project contains the package that you would like to update. Next, you will see a list of the packages that are currently installed in your project which have an update available. Choose which one you want to update. If no packages in your solution have an update available, you will see a message notifying you of this.
```

File: docs/csharp/package-management.md [hunk 6]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+
+Finally, you will be able to select which version you would like to update the package to from a dropdown list of all available versions.
+
+![alt text](images/package-management/nuget-command-update-versionselector.png)
+
+C# Dev Kit will then perform the necessary changes and update your project file and references.
+
+### Remove a package
+
+To remove a NuGet package from your project, use the command “NuGet: Remove NuGet Package.” If you have more than one project in your solution, you will be asked to select which project that you would like to remove a package from.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:56]:
```suggestion
To remove a NuGet package from your project, use the command **NuGet: Remove NuGet Package**. If you have more than one project in your solution, you will be asked to select which project that you would like to remove a package from.
```

File: docs/csharp/package-management.md [hunk 7]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+NExt, you will see a list of the packages that are currently installed in your project and let you choose which one you want to remove. C# Dev Kit will then perform the necessary changes and update your project file and references.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:57]:
```suggestion
Next, you will see a list of the packages that are currently installed in your project and let you choose which one you want to remove. C# Dev Kit will then perform the necessary changes and update your project file and references.
```

File: docs/csharp/package-management.md [hunk 8]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+
+* [NuGet Commands in C# Dev Kit](#nuget-commands-in-c-dev-kit)
+* [Dependency Management](#dependency-management)
+
+## NuGet Commands in C# Dev Kit
+
+In C# Dev Kit, you can perform NuGet package operations in either of two ways: commands in the command palette or through right-click actions in the C# Dev Kit solution explorer. The below documentation details how to use each NuGet command through both of the two methods.
+
+### Add a package
+
```
Comment (ntrogh) [inline docs/csharp/package-management.md:23]: You might consider using a numbered list to make these steps more scannable.

File: docs/csharp/package-management.md [hunk 9]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+![alt text](images/package-management\nuget-command-addpackage-search2.png)
+
+Next, select the version you want to apply.
+
+![alt text](images/package-management/nuget-command-addpackage-versionselection.png)
+
+Once you select a package and version number, C# Dev Kit will add it to your project and update your project file and references.
+
+### Update a package
+
```
Comment (ntrogh) [inline docs/csharp/package-management.md:45]: Similar, consider using a numbered list.

File: docs/csharp/package-management.md [hunk 10]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+To update a NuGet package in your project, use the command “NuGet: Update NuGet Package”. If you have more than one project in your solution, you will be asked to select which project contains the package that you would like to update. Next, you will see a list of the packages that are currently installed in your project which have an update available. Choose which one you want to update. If no packages in your solution have an update available, you will see a message notifying you of this.
+
+Finally, you will be able to select which version you would like to update the package to from a dropdown list of all available versions.
+
+![alt text](images/package-management/nuget-command-update-versionselector.png)
+
+C# Dev Kit will then perform the necessary changes and update your project file and references.
+
+### Remove a package
+
```
Comment (ntrogh) [inline docs/csharp/package-management.md:55]: Similar, consider using a numbered list.

File: docs/csharp/package-management.md [hunk 11]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+### Add a package
+
+To add a NuGet package to your project, use the command “NuGet: Add NuGet Package”.
+
+![alt text](images/package-management/nuget-command-addpackage.png)
+
+If you have more than one project in your solution, you will be asked to select which project you want to add the package to.
+
+![alt text](images\package-management\nuget-command-addackage-projectselection.png)
```
Comment (ntrogh) [inline docs/csharp/package-management.md:30]:
```suggestion
![alt text](images/package-management/nuget-command-addackage-projectselection.png)
```

File: docs/csharp/package-management.md [hunk 12]
```diff
@@ -10,7 +10,51 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+
+Then you can search for packages by name.
+
+![alt text](images/package-management/nuget-command-addpackage-search1.png)
+
+![alt text](images/package-management\nuget-command-addpackage-search2.png)
```
Comment (ntrogh) [inline docs/csharp/package-management.md:36]:
```suggestion
![alt text](images/package-management/nuget-command-addpackage-search2.png)
```

**Review by ntrogh — COMMENTED** [review]
@albarry4 Thanks for the update! Some formatting suggestions and ask to include alt text for images.

**Comment by albarry4** [comment]
@ntrogh I just finished making those suggested changes -- would you be able to review again?

File: docs/csharp/package-management.md [hunk 13]
```diff
@@ -10,7 +10,47 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+4. Next, select the version you want to apply.
+![Screenshot showing quickpicks dropdown menu with placeholder text that reads: "Select a NuGet package version". The quickpick options show a list of example NuGet package versions to choose from.](images/package-management/nuget-command-addpackage-versionselection.png)
+
+5. Once you select a package and version number, C# Dev Kit will add it to your project and update your project file and references.
+
+### Update a package
+
+1. To update a NuGet package in your project, use the command **NuGet: Update NuGet Package**.
+2. If you have more than one project in your solution, you will be asked to select which project contains the package that you would like to update.
+3. Next, you will see a list of the packages that are currently installed in your project which have an update available. Choose which one you want to update. If no packages in your solution have an update available, you will see a message notifying you of this.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:43]:
```suggestion
3. Next, choose which package to update from the list of packages that are currently installed in your project and that have an update available. If no packages in your solution have an update available, you see a message notifying you of this.
```

File: docs/csharp/package-management.md [hunk 14]
```diff
@@ -10,7 +10,47 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+4. Finally, you will be able to select which version you would like to update the package to from a dropdown list of all available versions.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:44]:
```suggestion
4. Finally, you can select which version you would like to update the package to from a dropdown list of available versions.
```

File: docs/csharp/package-management.md [hunk 15]
```diff
@@ -10,7 +10,47 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+![Screenshot showing quickpicks dropdown menu with placeholder text that reads: "Select a NuGet package version". The quickpick options show a list of example NuGet packages to choose from. There are indicators on the list to show which version is currently installed in the users project ("current"), and which is the latest available version ("latest")](images/package-management/nuget-command-update-versionselector.png)
+5. C# Dev Kit will then perform the necessary changes and update your project file and references.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:46]:
```suggestion
5. C# Dev Kit then performs the necessary changes and updates your project file and references.
```

File: docs/csharp/package-management.md [hunk 16]
```diff
@@ -10,7 +10,47 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+
+### Remove a package
+
+1. To remove a NuGet package from your project, use the command **NuGet: Remove NuGet Package**.
+2. If you have more than one project in your solution, you will be asked to select which project that you would like to remove a package from.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:51]:
```suggestion
2. If you have more than one project in your solution, you are asked to select which project that you would like to remove a package from.
```

File: docs/csharp/package-management.md [hunk 17]
```diff
@@ -10,7 +10,47 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+3. Next, you will see a list of the packages that are currently installed in your project and let you choose which one you want to remove.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:52]:
```suggestion
3. Next, choose which package you want to remove from the list of packages that are currently installed in your project.
```

File: docs/csharp/package-management.md [hunk 18]
```diff
@@ -10,7 +10,47 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+4. C# Dev Kit will then perform the necessary changes and update your project file and references.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:53]:
```suggestion
4. C# Dev Kit then performs the necessary changes and updates your project file and references.
```

**Review by ntrogh — CHANGES_REQUESTED** [review]
Suggested some minor rewording and formatting fixes. Otherwise, should be good to go.

**Comment by albarry4** [comment]
> Suggested some minor rewording and formatting fixes. Otherwise, should be good to go.

made all of those changes! :)

**Review by ntrogh — APPROVED** [review]

**Pull request merged**